### DISCUSSION TITLE
Support `styled` syntax like Emotion and Styled Components

### DISCUSSION THREAD

**Discussion by tihuan** [chakra-ui/panda#808] [Ideas]
It seems like Panda currently only supports creating styling objects through POJO, is there a plan to support `styled` syntax, so people can opt in to write css directly?

My thoughts are: 

1. Only allowing writing styles in POJO means users would need to wait for Panda CSS to add the new styling rules before they could use them even when certain browsers have already supported them
2. Some people just like to write CSS and not POJO
3. Easier to port large projects currently using styled syntax to Panda

Thank you!

**Comment by anubra266** [comment]
Perharps you mean this
https://github.com/chakra-ui/panda/pull/786

**Comment by tihuan** [comment]
Yaaas exactly!! Just subscribed to that PR 🔥 Thanks so much @anubra266 🙏
